src/index.js

```diff
@@ -1,13 +1,20 @@
 import './main.css';
 import clearListContainer from './utils/clear-container.js';
 import checkAndUncheckTodo from './utils/check-uncheck-todo.js';
+import clearAllCompletedTodos from './utils/clear-all-completed-todos.js';
+import editTodo from './utils/edit.js';
+// import deleteTodo from './utils/delete-todo.js';
```

Comment:
[OPTIONAL]

- I don't want to block you for this little change. But please remove this commented code before merging to make the codebase cleaner.

Reply:
Wow... That's okay bro. Thanks so much. Really appreciate that.